infrastructure/repository/customer_comments_repo.py

```diff
@@ -0,0 +1,24 @@
+from infrastructure.repository.base_repository import BaseRepository
+from infrastructure.models import UserComments, UserAccessDetails
+from datetime import datetime
+from sqlalchemy import exc
+
+
+class CustomerComments(BaseRepository):
+    def submit_comment(self, address, comment, email, name):
+
+        try:
+            user_comment = UserComments(wallet_address=address, comment=comment)
+            user_access_log = UserAccessDetails(
+                wallet_address=address,
+                email=email,
+                name=name,
+                login_time=datetime.now(),
+            )
+
```

Comment:
We don't need to add try and except here as add_item takes care of it.

Reply:
Fix in #4